src/System.Private.ServiceModel/tools/test/HelixLabRun/launchWCFOpenBuilds.cmd

```diff
@@ -0,0 +1,88 @@
+REM
+REM This script sync to the latest WCF source, build and submit Helix lab runs.
+REM Arguments:
+REM %1: Lab scripts location
+REM %2: WCF local git source location
+REM %3: Response files location
+
+@echo off
+setlocal
+
+set BridgeHost=
+pushd %2
+pushd bin\wcf\tools\Bridge
+if [%ERRORLEVEL%]==[0] (
+    call Bridge.exe -stop
+    popd
```

Comment:
We have a StopBridge.cmd script.  I recommend you use that for consistency.  It avoids this logic to pushd to the Bridge bin folder too.